docs/app/pages/components/sections/search/search-history-ng1/snippets/controller.js

```diff
@@ -1,8 +1,8 @@
 angular.module('app').controller('SearchHistoryDemoCtrl', SearchHistoryDemoCtrl);
 
-SearchHistoryDemoCtrl.$inject = ['$scope', '$modal'];
+SearchHistoryDemoCtrl.$inject = ['$scope', '$modal', '$persistentDataService'];
```

Comment:
Might be best to keep this example using localStorage as these changes break the codepen and for people only using Angular 1 and not a hybrid this code will not work.